### ISSUE TITLE
Introduce displaying of user details

### ISSUE THREAD

**Issue by mislavmatijevic** [foivz/pop-app#38]
Currently, after pressing on the "user" icon in the top-right corner of the _main menu_ screen, the following appears:

<img src="https://github.com/foivz/pop-app/assets/11699688/74bb1bcc-4f9d-44ee-9cbd-df2454810453" width="200">

The _Logout_ button is way too large and the _Cancel_ button makes no sense since this is no operation to be canceled. Exiting such "pop-ins" is usually done by pressing anywhere outside of it.

Since all user data is stored in the _User.loggedIn_ static attribute, it's a shame not to display this info to the user on demand. And this "user" icon is just the place for that!

This issue requires reworking what happens after the "user" icon is pressed. This issue proposes the following design.

_Of course, the proposed design doesn't have to be strictly followed. It's sole purpose is to aid the developer in implementing this feature._

## For buyer
![pop_app-buyer_drawer](https://github.com/foivz/pop-app/assets/11699688/d7407961-ab28-4aea-a63f-7a3c2c8d6f14)

## For seller
![pop_app-seller_drawer](https://github.com/foivz/pop-app/assets/11699688/870faf4f-ac92-4725-8e95-a52f7c0f9f0d)

**Comment by jfletcher20** [comment]
Agreed on the logout button size. I'm not sure why that ended up so large since I had adjusted it to be small and side-by-side; that was probably lost during merge.

I think UI menus look a lot better in the app's signature red than yellow;  then the text and icons are all white instead of black. The yellow looks ok as an accent sometimes, but I've come to like the blue/red combo a lot more otherwise (up to you though).

**Comment by jfletcher20** [comment]
After corresponding IRL, added another task:
- [x] remove profile icon from appbar and relocate the functionality to the main menu gridview instead of the settings tab which isn't being used

**Comment by mislavmatijevic** [comment]
I must be honest, I'm not sure what does the "IRL" abbrevation mean?

**Comment by jfletcher20** [comment]
In real life

**Comment by mislavmatijevic** [comment]
Oh. "Corresponding in real life"? Couldn't figure that one out.
